### DISCUSSION TITLE
value of type 'StorePublisher' has no dynamic member 'values' using key path from root type 'State'

### DISCUSSION THREAD

**Discussion by saroar** [pointfreeco/swift-composable-architecture#799] [Q&A]
**Describe the bug**
Give a clear and concise description of what the bug is.
using fastlane for my app and getting this error 
❌  /Users/alif/Library/Developer/Xcode/DerivedData/Addame-biwqafsqwbkxfaakpgvfcpnddibd/SourcePackages/checkouts/swift-composable-architecture/Sources/ComposableArchitecture/Beta/Concurrency.swift:222:10: value of type 'StorePublisher<State>' has no dynamic member 'values' using key path from root type 'State'
        .values
^
❌  /Users/alif/Library/Developer/Xcode/DerivedData/Addame-biwqafsqwbkxfaakpgvfcpnddibd/SourcePackages/checkouts/swift-composable-architecture/Sources/ComposableArchitecture/Beta/Concurrency.swift:221:22: subscript 'subscript(dynamicMember:)' requires that 'State' conform to 'Equatable'

**To Reproduce**
Zip up a project that reproduces the behavior and attach it by dragging it here.

```swift
// And/or enter code that reproduces the behavior here.
swift 5.5 
```
my project is open source here is it 
https://github.com/AddaMeSPB/AddaMeIOS/tree/feature/git_workflow

**Expected behavior**
Give a clear and concise description of what you expected to happen.

**Screenshots**

**Environment**
  - swift-composable-architecture version [0.27.0]
  - Xcode [e.g. 13 beta 5]
  - Swift [e.g. 5.5]
  - OS: [e.g. iOS 14]

**Additional context**
Add any more context about the problem here.

**Comment by iampatbrown** [comment]
What's the easiest way to reproduce the behaviour? 
Your project is pretty big. If you are not using the TCA concurrency features in your app, it's likely something to do with your fastlane config.
I wasn't able to run any of your lanes locally so if you can provide the steps to take a new system to this error that would help. 
Thanks.

**Comment by saroar** [comment]
oo thanks for your reply its say none of my lane was not working in your mac :( I think I am not using `TCA concurrency features` right now.

**Comment by stephencelis** [comment]
@saroar Curious if you're seeing this _locally_ with fastlane? Or is it a CI environment? Also, does the problem persist with the Xcode 13 RC that was released today?

**Comment by stephencelis** [comment]
Hi @saroar, I'm going to convert this to a discussion, as it doesn't seem to be a reproducible library bug and might have more to do with how Fastlane is configured. If we do discover it's a bug, we can reopen an issue with more info on how to reproduce.
